schema/patches/17.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE IF EXISTS public.terms
+    ADD COLUMN week_names text;
```

Comment:
May be better to store it as `JSONB` (or even `text[]`) to make the nature clearer? idk